author.rb

```diff
@@ -0,0 +1,19 @@
+class Author
+  attr_reader :id, :first_name, :last_name, :items
+
+  def initialize(_id, first_name = '', last_name = '')
+    @id = generate_random_id
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def generate_random_id
+    rand(1..100)
+  end
+
+  def add_item(item)
+    @items << item
+    item.author = self
+  end
```

Comment:
update, the add_item method in the Author class to takes an additional parameter game_library. When calling the add_item method, you can pass the game_library instance as an argument, and the author will be added directly to the authors array of the GameLibrary instance.
You can use the code below : 
` def add_item(item, game_library)
    @items << item
    item.author = self
    game_library.authors << self
  end`
That, the added authors will be stored automatically in the library.